### PR TITLE
Modified talent show page

### PR DESCRIPTION
**Pull request by CatalinaAlbornoz**

### CODE REVIEW

File: jinja/carnival/talent_show.html [hunk 1]
```diff
@@ -12,53 +12,64 @@
...
-            <img src="../img/carnival_meme_example.png" class="logo" style="max-width: 500px;"></img>
-        </div>
+            <div class = "row justify-content-center align-items-center">    
+                <div class="col">            
+                    <img src="../img/transpilation.jpeg" class="rounded mx-auto d-block" style="max-width: 300px;"></img>
+                    <figcaption style="text-align: center">Natchapol Patamawisut</figcaption>
+                </div>
+
+                <div class="col">            
+                    <img src="../img/ml-qml.jpg" class="rounded mx-auto d-block" style="max-width: 300px;"></img>
```
Comment (alvaro-at-xanadu) [inline jinja/carnival/talent_show.html:65]: Do we not have the name of this person?

File: jinja/carnival/talent_show.html [hunk 2]
```diff
@@ -12,53 +12,64 @@
...
-            <p>
-            Have you ever wanted to participate in the PennyLane community without writing code? Do you love our hand-drawn images and want to explore your artistic side too 🖌️?
-This is your chance! 
-</p>
+        </div>
+            
+            <div class = "row justify-content-center align-items-center">    
+                <div class="col">            
+                    <img src="../img/draw__pennylane.png" class="rounded mx-auto d-block" style="max-width: 300px;"></img>
+                    <figcaption style="text-align: center;">Alberto Maldonado Romo</figcaption>
```
Comment (alvaro-at-xanadu) [inline jinja/carnival/talent_show.html:24]: Is it OK to have a person win twice?

**Review by alvaro-at-xanadu — COMMENTED** [review]
I'm a bit puzzled by the current state of the main carnival page:
We have a separate sticker for the **Talent show winners**, but the winners have been put in the **Talent show page**. Should we just delete the Talent show and change it for Talent show winners?
There's also a special Announcements page that is unused, and the Carnival is basically over. Should we get rid of it? Or should we add something there?
Should we change the Trivia Party Page just with a message: Thank you to all who came, if you missed it, keep an eye out for future event etc. (something better than this). (This can wait)
For now the **urgent thing is posting the winners.** I think the page is fine except for the two questions I posted in the code. I also think we need to **decide what to do with the talent show stickers, so we don't have a useless one floating around**.

File: jinja/carnival/talent_show.html (hunk 1 above)
Comment (CatalinaAlbornoz) [reply jinja/carnival/talent_show.html:65]: No, we don't

File: jinja/carnival/talent_show.html (hunk 2 above)
Comment (CatalinaAlbornoz) [reply jinja/carnival/talent_show.html:24]: Yes, it's ok. He submitted two very good pieces.

**Comment by CatalinaAlbornoz** [comment]
> I'm a bit puzzled by the current state of the main carnival page: We have a separate sticker for the **Talent show winners**, but the winners have been put in the **Talent show page**. Should we just delete the Talent show and change it for Talent show winners? There's also a special Announcements page that is unused, and the Carnival is basically over. Should we get rid of it? Or should we add something there? Should we change the Trivia Party Page just with a message: Thank you to all who came, if you missed it, keep an eye out for future event etc. (something better than this). (This can wait) For now the **urgent thing is posting the winners.** I think the page is fine except for the two questions I posted in the code. I also think we need to **decide what to do with the talent show stickers, so we don't have a useless one floating around**.

Yes, we can probably remove the last 2 cards.

**Review by alvaro-at-xanadu — APPROVED** [review]
Ok, let's do this!

**Review by josh146 — APPROVED** [review]

**Pull request merged**